### PR TITLE
Add Optimism and fix Avalanche C chain

### PR DESCRIPTION
**Pull request by hewigovens**

### CODE REVIEW

File: blockchains/avalanchec/assets/0x60781C2586D68229fde47564546784ab3fACA982/info.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+    "name": "Pangolin",
+    "website": "https://pangolin.exchange/",
+    "description": "A community-driven decentralized exchange for Avalanche and Ethereum assets with fast settlement, low transaction fees, and a democratic distribution–powered by Avalanche",
+    "explorer": "https://cchain.explorer.avax.network/address/0x60781C2586D68229fde47564546784ab3fACA982",
+    "research": "https://research.binance.com/en/projects/avalanche",
+    "type": "AVALANCHE-ERC20",
```
Comment (vikmeup) [inline blockchains/avalanchec/assets/0x60781C2586D68229fde47564546784ab3fACA982/info.json:7]: Just `AVALANCHE` ?

File: blockchains/optimism/assets/0x8700dAec35aF8Ff88c16BdF0418774CB3D7599B4/info.json [hunk 2]
```diff
@@ -0,0 +1,38 @@
+{
+    "name": "Synthetix Network",
+    "website": "https://synthetix.io",
+    "description": "Synthetix is a derivatives liquidity protocol on Ethereum that enables the issuance and trading of synthetic assets.",
+    "explorer": "https://optimistic.etherscan.io/address/0x8700daec35af8ff88c16bdf0418774cb3d7599b4",
+    "research": "https://research.binance.com/en/projects/synthetix",
+    "type": "OPTIMISM-ERC20",
```
Comment (vikmeup) [inline blockchains/optimism/assets/0x8700dAec35aF8Ff88c16BdF0418774CB3D7599B4/info.json:7]:
```suggestion
    "type": "OPTIMISM",
```

Comment (vikmeup) [reply blockchains/optimism/assets/0x8700dAec35aF8Ff88c16BdF0418774CB3D7599B4/info.json:7]: This one is tricky, which format we should follow here

File: blockchains/avalanchec/assets/0x60781C2586D68229fde47564546784ab3fACA982/info.json (hunk 1 above)
Comment (chris-zedio) [reply blockchains/avalanchec/assets/0x60781C2586D68229fde47564546784ab3fACA982/info.json:7]: or we can do symbol-20? AVAX20

File: blockchains/optimism/assets/0x8700dAec35aF8Ff88c16BdF0418774CB3D7599B4/info.json (hunk 2 above)
Comment (chris-zedio) [reply blockchains/optimism/assets/0x8700dAec35aF8Ff88c16BdF0418774CB3D7599B4/info.json:7]: similar to polygon, optimism may have other token types

File: blockchains/avalanchec/assets/0x60781C2586D68229fde47564546784ab3fACA982/info.json (hunk 1 above)
Comment (hewigovens) [reply blockchains/avalanchec/assets/0x60781C2586D68229fde47564546784ab3fACA982/info.json]: done

**Pull request merged**